Emby.Server.Implementations/HttpServer/HttpListenerHost.cs

```diff
@@ -655,11 +643,6 @@ private void RedirectToSecureUrl(IHttpRequest httpReq, IResponse httpRes, string
             }
             else
             {
-                // TODO what is this?
-                var httpsUrl = url
-                    .Replace("http://", "https://", StringComparison.OrdinalIgnoreCase)
-                    .Replace(":" + _config.Configuration.PublicPort.ToString(CultureInfo.InvariantCulture), ":" + _config.Configuration.PublicHttpsPort.ToString(CultureInfo.InvariantCulture), StringComparison.OrdinalIgnoreCase);
-
```

Comment:
was that broken, or was that just dead code?

Reply:
Dead, the var is never used.